cid/helpers/quicksight/dashboard.py

```diff
@@ -53,7 +65,8 @@ def latest(self) -> bool:
 
     @property
     def latest_version(self) -> int:
-        return self.sourceTemplate.version
+        if isinstance(self.sourceTemplate, CidQsTemplate):
+            return self.sourceTemplate.version
```

Comment:
What if not an instance of `CidQsTemplate`? Should we return `None` or `-1`?

Reply:
No sure what is the purpose of -1 but we we can do that for consistency